_codux/boards/intro/intro.board.tsx

```diff
@@ -14,7 +14,7 @@ export default createBoard({
                 create your own store easily. Use Codux to design your website the way you want, and
                 Wix Headless e-commerce services to manage your store.
                 <a href={'https://help.codux.com/kb/en/article/kb37137'}>
-                    <button className={CommonStyles_module['intro-button']}>Learn more</button>
+                    <button className={commonStyles['intro-button']}>Learn more</button>
```

Comment:
I would suggest we unify rules casing then too.

Reply:
fixed